src/deltacode/__init__.py

```diff
@@ -67,6 +85,48 @@ def __init__(self, new_path, old_path, options):
             self.deltas.sort(key=lambda Delta: Delta.factors, reverse=False)
             self.deltas.sort(key=lambda Delta: Delta.score, reverse=True)
 
+    def get_files(self,codebase,is_new):
+        """
+        Walk through the codebase, then generate the resources it(including all files and its directories)
+        then we enumerate over this generated codebase to get file, and directories as (obj)
+        Now during the time of enumeration we append files in the self.new_files list and incremants out self.new_files_count
+        Similarly for old_files.
+        Now , we also maintain a map which maps from object path to its fingerprint.
+        This map will be required when we calculate the hamming distances and compare similarity.
+        """
+        resources = codebase.walk_filtered(topdown=True)
+        for i,obj in enumerate(resources):            
+            if is_new:
+                # append in the new_files
+                self.new_files.append([obj,''])
+                try :
+                    self.new_files_fingerprint[obj.path] = obj.fingerprint
+                except AttributeError:
+                    self.new_files_fingerprint[obj.path] = None
+                if obj.is_file:
+                    # increment the new files count
+                    self.new_files_count += 1
+
+            else:
+                # append in the old files
+                self.old_files.append([obj,''])
+                try:
+                    self.old_files_fingerprint[obj.path] = obj.fingerprint
+                except AttributeError:
+                    self.old_files_fingerprint[obj.path] = None
+                if obj.is_file:
+                    # increment the old files count
+                    self.old_files_count += 1
+                    
+
+    def enumerate_files_from_codebases(self):
+       """
+       An method which call the utility function get_files for generating the codebase
+       """
+       self.get_files(self.codebase1,is_new = True)
+       self.get_files(self.codebase2,is_new = False)
```

Comment:
This function's name is unrelated to what it actually does.

Reply:
@majurg , I think interchanging the function names `get_files` and `enumerate_files_from_codebases` would better match with the situation

Reply:
changed the function name